tests/actions/test_models.py

```diff
@@ -14,6 +14,28 @@ def action():
     )
 
 
+def test_versions_are_ordered_from_latest_to_oldest(action):
```

Comment:
Let's use the language that Django uses. So, either first/last or latest/earliest. Alternatively, let's use newest/oldest.

```suggestion
def test_versions_are_ordered_from_latest_to_earliest(action):
```

Reply:
Changed in facf591f765f4548efabff008b8fcc791aaf8fad